packages/dashboard/src/components/pageStructure/index.js

```diff
@@ -245,22 +245,31 @@ export function LeftRail() {
             {enabledPrimaryPaths.map(({ Icon, ...path }) => {
               const isNotificationBubbleEnabled =
                 path.value === APP_ROUTES.TEMPLATES_GALLERY;
+              const appendNewBadgeToLable = (label) =>
+                isNotificationBubbleEnabled
+                  ? sprintf(
+                      /* translators: 1: current page. 2: number of new items. */
```

Comment:
thanks!

Reply:
Probs also only if there's a number in the notification bubble - will fix the karma test